cli/src/helpers/logNextSteps.ts

```diff
@@ -35,7 +37,7 @@ export const logNextSteps = async ({
   }
 
   if (["postgres", "mysql"].includes(databaseProvider)) {
-    logger.info("  ./start-database.sh");
+    logger.info(`  ./start-database.sh (${chalk.red("Optional")})`);
```

Comment:
You **do need** a db so maybe it should be like "ensure db running. You can start one using ./start-database.sh" although thats kinda long.

Reply:
This might be it - "start db, if necessary. Use ./start-database.sh" and I can put red around the "if necessary" part.

Reply:
Or "If db is down, use ./start-databse.sh"

Reply:
![image](https://github.com/t3-oss/create-t3-app/assets/63444697/286f624d-bb93-4ae7-86d1-53e1854af974)